src/Program.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Threading.Tasks;
+using DSharpPlus;
+using DSharpPlus.CommandAll;
+using DSharpPlus.CommandAll.Parsers;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using NotifierRedirecter.Events;
+using Serilog;
+using Serilog.Events;
+using Serilog.Sinks.SystemConsole.Themes;
+
+namespace NotifierRedirecter;
+
+public sealed class Program
+{
+    internal static readonly IServiceCollection Services;
+    internal static readonly IConfiguration Configuration;
+    internal static readonly Database Database;
+    internal static readonly ILoggerFactory LoggerFactory;
+    private static readonly ILogger<Program> Logger;
+
+    static Program()
+    {
+        Configuration = new ConfigurationBuilder()
+            .AddJsonFile("config.json", true, true)
```

Comment:
I might be blind. But I don't see a example for a config.json. If a example config does not exist. I would prefer to have one!

Reply:
[`res/config.json`](https://github.com/OoLunar/NotifierRedirecter/blob/main/res/config.json)

Reply:
Ah there. Yea I was blind!